client/src/app/pages/main/footer/footer.component.html

```diff
@@ -58,6 +58,12 @@
               <div>Docs</div>
             </a>
           </div>
+          <div class="mt-4 mx-1 link">
```

Comment:
Yeah let's add this closer to the steam link near the left, it can be the second icon.